docs/dev/serialization.md

```diff
@@ -81,27 +81,35 @@ If the returned object has a `_from_json_dict_` attribute, it is called instead.
 
 All of Cirq's public classes should be serializable.
 This is enforced by the `test_json_test_data_coverage` test in
-`cirq/protocols/json_serialization_test.py`, which iterates over cirq's API looking for types
```

Comment:
Thank you for writing this up - I am working on a better documentation and fixing some of the issues (e.g. you won't have to modify the `imports` dict, and I'm adding a new `modules.md` doc for setting up a new module). We can remove these changes from this PR.